### ISSUE TITLE
Opinions on Providing a Fake for CIDR/IP Prefixes

### ISSUE THREAD

**Issue by MichaelCrowcroft** [FakerPHP/Faker#796]
Before submitting a PR keen to get thoughts on whether a method to provide CIDR blocks/IP Prefixes would make sense for this package.

[Thinking something basic like the Ruby package has.](https://github.com/faker-ruby/faker/commit/3c53882e3524a7df395ac7b12fe3c46bea3f2aee)

Appreciate this fairly simple to implement in projects with the existing methods, but feels like it would be nice to have.

**Comment by pimjansen** [comment]
I agree that it could be a good thing though we are not adding new features to the 1.x.

@localheinz @bram-pkg i think we can include cidr blocks in the core packages right?

**Comment by bram-pkg** [comment]
This sounds like an ideal core package candidate, yep.
